defaults/test_default.yaml

```diff
@@ -1,3 +1,6 @@
+append_scylla_yaml:
```

Comment:
add link to issue why we need that

Reply:
Done

Reply:
btw. I think this means initial tablets will be 2*2^10 isn't it? (see `configurations/tablets-initial-32.yaml` where 4 means 32 tablets).
This approach also is not taking into account number of shards, but for the sake of quick workaround might be good.

Reply:
@pehala we were explicitly told not to use this parameter, and use the initial_tablets inside the create keyspace command.
What has changed?

Reply:
I just realized, this param changed its purpose... https://github.com/scylladb/scylladb/commit/f043c83ba539379411a7b7e48b964da649abcef9

Reply:
and this change didn't yet reached 2025.1 branch.